src/components/SingleBlog/index.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState, useEffect } from "react";
+import "./index.scss";
+import { Container, Col, Row } from "react-bootstrap";
+import { useParams } from "react-router-dom";
+import { ReactComponent as FacebookIcon } from "../../images/icons/facebookIcon.svg";
+import { ReactComponent as TwitterIcon } from "../../images/icons/twitterIcon.svg";
+import { ReactComponent as LinkedInIcon } from "../../images/icons/linkedInIcon.svg";
+import { ReactComponent as InstagramIcon } from "../../images/icons/instagramIcon.svg";
+
+const RELATED_POSTS_TO_SHOW = 4;
+
+export default function SingleBlog(props) {
+  const params = useParams();
+  const [blogPost, setBlogPosts] = useState();
+  const allBlogPosts = async () => {
+    const data = await fetch(
+      `https://public-api.wordpress.com/wp/v2/sites/safeplace102505649.wordpress.com/posts/${params.id}`
+    );
+    const posts = await data.json();
+    setBlogPosts(posts);
+  };
+  useEffect(() => {
+    allBlogPosts();
+  });
```

Comment:
, [] in useEffect is missing

Reply:
no, it's not, it has to be removed in order to avoid the warning message bellow 
![screenshot1](https://user-images.githubusercontent.com/62549249/99846956-55cd5a00-2b88-11eb-9238-1d4c9010f923.jpeg)

Reply:
you can try to move the function declaration inside of useEffect, because normally useEffect needs a second parameter. maybe @louisrli can advise better

Reply:
google the error or read what the warning says please